.travis_install.sh

```diff
@@ -43,7 +43,8 @@ elif [[ "${ARCH}" = "linux32x86" ]]; then
             libaudio-dev:i386 \
             gcc-multilib \
             uuid-dev:i386 \
-            libcurl3-dev:i386
+            libcurl4-openssl-dev:i386 \
+			libssh2-1-dev:i386
```

Comment:
Are installing:

- libssh2-1-dev:i386
- libcurl4-openssl-dev:i386

an artefact of trying to figure out this issue, or really required?  If they are required, should the 64 bit libraries be added as well (and why isn't the 64 bit version of libcurl3 loaded? :-))

Reply:
uhm, yes... those are remainders of me trying to solve the problem. 
probably not needed :)